src/merge.rs

```diff
@@ -0,0 +1,138 @@
+use std::marker;
+use std::mem;
+use libc::c_uint;
+
+use {raw, Object, FileFavor};
+use util::Binding;
+use call::Convert;
+
+/// annotated commits, the input to merge and rebase
+pub struct AnnotatedCommit<'repo> {
+    raw: *mut raw::git_annotated_commit,
+    _marker: marker::PhantomData<Object<'repo>>,
```

Comment:
Also, could some accessor functions of `AnnotatedCommit` also be exposed? It looks like there's a few simple-ish ones in the header files